internal/auth/auth_test.go

```diff
@@ -31,11 +32,13 @@ func TestCreateAccount(t *testing.T) {
 
 	for name, tt := range tests {
 		t.Run(name, func(t *testing.T) {
-			originalCreateAccount := createAccount
+			defer func() {
```

Comment:
`t.Fatal` calls FailNow, which runs runtime.Goexit - Goexit will run all defers before halting execution. If this test were to fail, this wouldn't be reset.. Now it will be :D

Reply:
Good catch

Reply:
Ah crap - good catch. Here's the bug you've pointed out:
https://play.golang.org/p/0KqD4o0IcSk

Here it's fixed:
https://play.golang.org/p/XwmcSSySjIi